selfdrive/car/ford/carcontroller.py

```diff
@@ -1,10 +1,10 @@
 from cereal import car
 from opendbc.can.packer import CANPacker
-from openpilot.common.numpy_fast import clip
 from openpilot.selfdrive.car import apply_std_steer_angle_limits
 from openpilot.selfdrive.car.ford import fordcan
 from openpilot.selfdrive.car.ford.values import CarControllerParams, FordFlags
 from openpilot.selfdrive.car.interfaces import CarControllerBase, V_CRUISE_MAX
+from openpilot.selfdrive.car.numpy_fast import clip
```

Comment:
it's probably time to just use numpy. I doubt this makes any difference at this point

Reply:
Possibly. I think this was also used because capnp couldn't automatically type coerce numpy floats/ints:

```
capnp.lib.capnp.KjException: Tried to set field: 'logMonoTime' with a value of: '1' which is an unsupported type: '<class 'numpy.int64'>'
```

Reply:
dataclasses don't explicitly type check or convert, so we will still need to explicitly convert somewhere. And it will get annoying and error prone if we forget this somewhere, and some capnp field is conditionally set in openpilot.

I think some sort of helper class is still needed for the time being, perhaps with a better name